dict/P/pandunia.tsv

```diff
@@ -663,6 +668,7 @@ PWN:europium.n.01		europium	europ·ium	eng:europium, fra:europium, spa:europio,
 PWN:even.r.01		hata	hata	ara: حَتَّى (ḥatta), fas:حَتّی, tur:hau:hatta, swa:hata, ful:haa, spa:hasta, por:até
 PWN:example.n.01		misal	misal	ara:مِثَال, may:misal, tur:misal, fas:مِثال, hin:मिसाल
 PWN:excellent.s.01		top klasik	top klas·ik	
+PWN:excessively.r.01		tai	tai	cmn:太 (tài)

```

Comment:
The word *tai* has already another definition, `PWN:extreme.s.01`. It is fine to use _tai_ in this sense too, because the source word from Chinese covers both senses too. In that case the etymology is not needed.

We can add later a synonym for this sense that is more cumbersome but also more precise, just like there are comparable synonyms in English (_too_ and _excessively_) and other languages.